src/runtime/wasm-tools.ts

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I didn't put it there because I wasn't sure if it's going to pull in a lot of unneeded dependencies into the runtime bundle. Since toProtoFile() was for cli tools not runtime and relies on src/cli/targets/... package, which is mostly needed for the command like pbjs/ts, I thought keeping it separate makes it easier to exclude and track.

Reply:
Fair enough.